kernel/data-pipeline/dialect/mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java

```diff
@@ -157,6 +158,7 @@ public synchronized void subscribe(final String binlogFileName, final long binlo
         dumpBinlog(binlogFileName, binlogPosition, queryChecksumLength());
         log.info("subscribe binlog file: {}, position: {}", binlogFileName, binlogPosition);
         reconnectTimes.set(0);
+        running = true;
```

Comment:
Could we put `reconnectTimes` and `running` reset logic to `connect` method instead of `subscribe` method?
Since they're related to connection but not subscription.